docs/api/index.rst

```diff
@@ -12,22 +12,8 @@ describing what they are and what they do. The APIs are grouped into the followi
 
       Automatically train/deploy predictors with 1 line of code for text, image, or tabular data.
 
-   .. card::
-      :title: Search Space
-      :link: autogluon.core.space.html
-
-      Hyperparameter search spaces of candidate hyperparameter values to consider.
```

Comment:
I don't think we should remove Spaces documentation: the existing documentation is useful: https://auto.gluon.ai/stable/api/autogluon.core.space.html#module-autogluon.core.space

Reply:
Removing as it referenced configspace in docs. We can add back afterwards in follow-up.